### DISCUSSION TITLE
[Feature Request]: Herd Pro logs custom folder

### DISCUSSION THREAD

**Discussion by JoostWan** [beyondcode/herd-community#352] [Ideas]
### Feature Description

We would like to add custom log locations for our Magento 2 projects.

**Comment by sschlein** [comment]
Aside from the logs, does M2 work properly? I haven't set up a test project yet because I don't have OpenSearch locally.

**Comment by JoostWan** [comment]
@sschlein Magento 2 works fine here! It only doesn't work on PHP 7.4, but that's a thing of the past anyway. You indeed need Opensearch, but it's easy to install via Brew."

**Comment by sschlein** [comment]
Answering your initial question, Herd doesn't support custom log paths at the moment, but that's definitely feature that we consider soon
